doc/source/train/user-guides/using-gpus.rst

```diff
@@ -104,6 +104,34 @@ You can get a list of associated devices with :meth:`ray.train.torch.get_devices
     trainer.fit()
 
 
+Setting the GPU type
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Ray Train allows you to specify the accelerator type for each worker.
+This is useful if your model training has some GPU memory constraints that requires a specific type of GPU.
+In a heterogeneous Ray cluster, this means that your training workers will be forced to run on the specified GPU type, 
+rather than on any arbitrary GPU node.
+
+For example, you can specify `accelerator_type="A100"` in the :class:`~ray.train.ScalingConfig` if you want to 
+assign each worker a NVIDIA A100 GPU.
+Ensure that your cluster has instances with the specified accelerator type 
+or is able to autoscale to fulfill the request.
```

Comment:
We can make this a tip:

```
.. tip::
    Ensure that your cluster has instances with the specified accelerator type 
or is able to autoscale to fulfill the request.
    Otherwise, your job will hang forever due to unsatisfiable pending resource requests.
```